bot.py

```diff
@@ -112,13 +124,119 @@ def on_message(ws, message):
         if admin_message(response):
             return
 
-    elif "team_join" == response["type"]:
-        if NEWUSERGREETING:
-            new_user_message(response)
-        USER_LIST = sc.api_call(  # noqa: 841
-            "users.list",
-        )
 
+    #TODO: Purge all this shit
+    # elif "team_join" == response["type"]:
+    #     if NEWUSERGREETING:
+    #         new_user_message(response)
+    #     USER_LIST = sc.api_call(  # noqa: 841
+    #         "users.list",
+    #     )
+
+    log(response)   
+
+def log(response):
+    global engine
+    def add_user(r):
+        q = engine.execute("SELECT last_name FROM users WHERE slack_id=%s", r["user"])
+        if q.fetchall() == []:
+            r = sc.api_call(
+                "users.info",
+                user=r["user"]
+                )
+            is_admin = r["user"]["is_admin"]
+            if is_admin:
+                is_admin = 1
+            else:
+                is_admin = 0
+            engine.execute("INSERT INTO users (slack_id, first_name, last_name, is_admin, is_pb_admin) VALUES (%s, %s, %s, "+str(is_admin)+", 0)", r["user"]["id"], r["user"]["profile"]["first_name"], r["user"]["profile"]["last_name"])
+
+    def add_channel(r):
+        q = engine.execute("SELECT name FROM channels WHERE slack_id=%s", r["channel"])
+        if q.fetchall() == []:
+            print 'callin'
```

Comment:
Typo? Also if you're logging somthing please follow @braxton-mclean's logging convention, look through the code for examples